### PR TITLE
fix: wrong translation key names

### PR DESCRIPTION
**Pull request by mprangenberg**
## Summary
**Fixed**
- Two wrong translation key names in calls from commands
**Changed**
- Added language test to check for translation calls that return the original key names, which means they are either missing or have a typo.

### PR Type

- [ ] This PR is a code change that implements a feature request.
- [X] This PR fixes an issue.
- [X] This PR adds a new feature that is not an open feature request.
- [ ] This PR is not a code change (e.g. documentation, README, ...)
### Other
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [X] If code changes were made then they have been tested.
- [ ] I have updated the documentation to reflect the changes.

### CODE REVIEW

File: test/lang.test.js [hunk 1]
```diff
@@ -23,39 +25,39 @@ describe('Localisation', function() {
 			locale = {};
 		}
 
-		describe(`# Language: ${lang.toUpperCase()}`, function() {
+		describe(`# Language: ${lang.toUpperCase()}`, function () {
```
Comment (Stefouch) [inline test/lang.test.js:28]: Your linter adds a space between `function` and `()` its parentheses. Could you remove these extra spaces?

**Review by Stefouch — CHANGES_REQUESTED** [review]
I love your new method to test every `__()` translation functions from all JS files. It's clever!

### To Fix
- [ ] Lint conflict

File: test/lang.test.js (hunk 1 above)
Comment (Stefouch) [reply test/lang.test.js]: I have pushed a fix myself. Is it OK for you?

Comment (mprangenberg) [reply test/lang.test.js]: Oh, that's ok. I just ran a format over the file. VS Code wants the code to look like this 😅

Comment (mprangenberg) [reply test/lang.test.js]: This method should find almost all calls. There are some instances it won't find because the key is build dynamically. But those are some small edge cases.

**Review by Stefouch — APPROVED** [review]
